frontend/src/components/CheckInComponent.jsx

```diff
@@ -14,20 +20,28 @@ function CheckInComponent() {
     setIsIn(false);
     setShow(false);
   }
+  // if cookie is already set
+  const cookieSet = () => {
+    if(myCookie){
+      setIsIn(true);
+    }
+  }
   const handleClick = () => {
     setIsIn(!isIn); 
     !isIn && setShow(true)
+    if (isIn) {
+      Cookies.set('userId', null);
```

Comment:
When the user, add a DELETE fetch request with the body being:
```js
{
  "userId": userId
}
```
before you remove the cookie.